### ISSUE TITLE
Compatibility with Vray Next

### ISSUE THREAD

**Issue by mOoOm** [Thunderloom/ThunderLoom#23]
Greetings,
I've tested to use Thunderloom with Vray 3.6 for my recent project, and it worked very well 
and it was so impressive. This tool enable artists to create entirely new and believable fabric texture.
But it doesn't seems to work well with Vray 4.x (aKa Vray Next), There were error log like,  'BRDFThunderloom shader cannot be loaded.'

**Comment by vidarn** [comment]
Hi mOoOm,
thanks for the kind words, and for bringing this problem to our attention. Are you using 3ds Max or Maya?
Vidar

**Comment by mOoOm** [comment]
I am using Maya. 🙂

**Comment by vidarn** [comment]
Ok, cool. There were some API changes in vray next, which is why the plugin does not work there. I have started updating the 3ds max plugin since I had everything set up for it already, but I will take care of the maya plugin next. Would you be willing to test it once I have something running? Are you on windows or a mac?

**Comment by vidarn** [comment]
Nice to meet you too Derek, I know you have put a lot of effort into testing Thunderloom and both Peter and I are very grateful for it :)
Here's a first test of thunderloom compiled for V-Ray next on maya2018 (windows): [V-Ray Next for Maya2018_v01.zip](https://github.com/Thunderloom/ThunderLoom/files/2698193/V-Ray.Next.for.Maya2018_v01.zip), please let me know if there are any problems!

**Comment by mOoOm** [comment]
Hi guys, thanks for super fast updating. :) Since I'm running Vray on same environment as Derek's, so there're no problem to test with the attachment files. I'll update you very soon.

**Comment by vidarn** [comment]
I’m sorry to hear that the plugin disappeared!
The only file that was actually updated in the zip file I uploaded was vray_thunderloom.dll so perhaps you could reinstall thunderloom 0.93 from the releases tab and verify that it shows up amongst the plugins again. Then you could try to just replace vray_thunderloom.dll. 
Sorry for the trouble!
Vidar

**Comment by mOoOm** [comment]
Hey Vidar and Peter. Just as my expect, the issue was occurred by my mistake - I had edited .mod file path as "+ VRayThunderLoomMtl ../thunderloom_maya_module", missing "any".  Your advices  were so helpful - because it helped me so sure that there was no issue those files themselves. Thanks.

To put the conclusion of test first - it works very well. I'm adding the screenshot so check it out.
But during the test there was a small error :  first time I tried to load plugin and there was an error log  "Unable to dynamically load : plug-ins/thunderLoomVRayMaya.mll" and I found out vray_thunderLoom.exp and lib files should be moved to "vrayplugins/RelWithDebInfo". In the zip file you've shared, those files are located with the .dll files. It was different from the file tree of  thunderloom 0.93.

Anyway, thanks for you guys effort I can keep working with this great plugin. I'm so happy. 
I hope my report could help you too.



![test_thunderloom_scarf](https://user-images.githubusercontent.com/6174540/50381693-b5870600-06d0-11e9-97f5-805e19bf61d6.jpg)
